### ISSUE TITLE
Replication Technique demonstrate target misleads opponents.

### ISSUE THREAD

**Issue by johnmeat** [magefree/mage#9543]
I cast [[Replication Technique]] I choose a target, I then choose to demonstrate and copy it and allow an opponent to copy the spell. The issue is that lets say I copy my [[Propaganda]], then the copy my opponent gets is also targeting my Propaganda. This is an illegal target (card is copy target permanent you control), so the opponent should change to a permanent that is their own, but more times than not they don't. The card itself functions correctly (i.e. the copy fizzles as the target is illegal), but I can see why my opponents get confused.

**Comment by Alex-Vasile** [comment]
Yeah that's an issue. Forget misleading, they should not be allowed to pick your permanents as a target in the first place.

**Comment by johnmeat** [comment]
Just to be clear they are not choosing to target my permanent, their copy is defaulting to targeting my permanent then they are not retargeting thus it fizzles.

**Comment by Alex-Vasile** [comment]
I imagine some of the other cards have a similar issue.

Without looking at the code, I imagine [[Excavation Technique]] could violate range of influence
